app/helpers/draws_helper.rb

```diff
@@ -22,6 +22,14 @@ def diff_class(diff)
     'zero'
   end
 
+  def lock_intent_btn_str(draw)
+    if draw.intent_locked
+      'Allow students to update their intent'
```

Comment:
These are _really_ long button labels - can we make them more concise and add tooltips instead?